front/app/modules/commercial/content_builder/services/ContentBuilder.ts

```diff
@@ -0,0 +1,48 @@
+import { API_PATH } from 'containers/App/constants';
```

Comment:
A bit unsure about the appropriate variable names for everything. If you have a different suggestion, just let me know. :)

Reply:
Not sure which variables you mean here - can you clarify?

Reply:
@IvaKop Just sort of in general I think :) I wasn't sure if things should be named "BuilderLayout" or "BuilderContent" or "ContentBuilderLayout" etc.